### ISSUE TITLE
Undefined refs and citations, article.bbl not found

### ISSUE THREAD

**Issue by zrimseku** [ReScience/MLRC#91]
I submitted a pull request #90 that has build issue - undefined refs and citations and article.bbl file is not found.

Log when errors begin:

```
Running 'xelatex -interaction=nonstopmode  -recorder  "article.tex"'
------------
Latexmk: Found input bbl file 'article.bbl'
Latexmk: Log file says output to 'article.pdf'
Latexmk: Found biber source file(s) [article.bcf bibliography.bib]
Build complete
Building zrimsek2022learning/
/home/andraz/MLRC/2021/zrimsek2022learning/journal
Latexmk: This is Latexmk, John Collins, 26 Dec. 2019, version: 4.67.
Latexmk: This is Latexmk, John Collins, 26 Dec. 2019, version: 4.67.
Rule 'pdflatex': The following rules & subrules became out-of-date:
      'pdflatex'
------------
Run number 1 of rule 'pdflatex'
------------
------------
Running 'xelatex -interaction=nonstopmode  -recorder  "article.tex"'
------------
Latexmk: References changed.
Latexmk: Non-existent bbl file 'article.bbl'
 No file article.bbl.
Latexmk: References changed.
Latexmk: References changed.
Latexmk: Log file says output to 'article.pdf'
Latexmk: List of undefined refs and citations:
  Citation 'chen2019pipr' on page 1 undefined on input line 9
  Citation 'chen2019pipr' on page 3 undefined on input line 50
  Citation 'gui2019dnn' on page 3 undefined on input line 50
  Citation 'hashemifar2018predicting' on page 3 undefined on input line 50
  Citation 'lv2021learning' on page 1 undefined on input line 3
  Reference `generalize' on page 7 undefined on input line 184
  Reference `graph' on page 7 undefined on input line 205
  Reference `graph' on page 9 undefined on input line 240
  Reference `subsets' on page 5 undefined on input line 104
  Reference `subsets' on page 6 undefined on input line 140
  Reference `subsets' on page 8 undefined on input line 231
  Reference `times' on page 6 undefined on input line 115
------------
Running 'make "article.bbl"'
------------
make[1]: *** No rule to make target 'article.bbl'.  Stop.
Latexmk: Summary of warnings from last run of (pdf)latex:
  Latex failed to resolve 7 reference(s)
  Latex failed to resolve 5 citation(s)
Collected error summary (may duplicate other messages):
  pdflatex: Command for 'pdflatex' gave return code 1
      Refer to 'article.log' for details
Latexmk: Use the -f option to force complete processing,
 unless error was exceeding maximum runs, or warnings treated as errors.
make: *** [Makefile:30: article.pdf] Error 12
Build complete
rm: cannot remove 'article.bbl': No such file or directory
rm: cannot remove 'article.blg': No such file or directory
```

**Comment by jeroenwijnen98** [comment]
@zrimseku, did you manage to solve this? We've run into a similar issue

**Comment by zrimseku** [comment]
Hi, yes I just did. There wasn't any problem with bibliography, but there were some with a table and with enumerating. In packages, I defined a new command, but this wasn't working anymore. I was also using enumerate package to get enumeration in form (1). When I changed those things I still get these reference and citation warnings, but now my pdf looks nice - all citations are there.

**Comment by Divyasha02** [comment]
@zrimseku how did you manage to solve it? could you please eloborate?

**Comment by zrimseku** [comment]
I deleted package `enumerate` and changed the enumerations in default mode, not (1) like I wanted to have.
I also enclosed every cell in table in math mode, because the new command that would take care of it automatically wasn't working. Check out my pull request #90 for more info.
Before I was searching for some problems with bibliography, but I think this error message is not relevant at all. Check build logs, you will find what is wrong with latex compilation there.
